homeassistant/components/sensor/blockchain.py

```diff
@@ -0,0 +1,55 @@
+"""
+Support for Blockchain.info sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.blockchain/
+"""
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+import voluptuous as vol
+
+REQUIREMENTS = ['python-blockchain-api==0.0.1']
```

Comment:
The `bitcoin` sensor is using the [blockchain](https://github.com/blockchain/api-v1-client-python) module. Perhaps it's worth to re-use that to avoid two deps while using both sensors..